tests/unit_tests/service/test_interface.py

```diff
@@ -436,6 +440,32 @@ def test_setup(mock_stomp: MagicMock):
     assert interface.worker()._ctx is not None
     assert interface.context().run_engine.scan_id_source == interface._update_scan_num
 
+    clear_path_provider()
+    interface.teardown()
+
+
+@patch("blueapi.service.interface.StompClient")
+def test_setup_without_path_provider_with_numtracker_makes_start_document_provider(
+    mock_stomp: MagicMock,
```

Comment:
Should: Is this needed? Given the config provided all stomp settings should be ignored anyway

Reply:
I've removed it from the other one too.